chapters/annotations.tex

```diff
@@ -1504,8 +1506,9 @@ \subsection{Version Handling}\label{version-handling}
 
 \subsubsection{Conversion rules}\label{conversion-rules}
 
+% Using mbox to avoid having line starting with ","
 There are a number of functions: \lstinline!convertClass!, \lstinline!convertClassIf!,
-\lstinline!convertElement!, \lstinline!convertModifiers!, \lstinline!convertMessage! defined as follows. The
+\lstinline!convertElement!, \mbox{\lstinline!convertModifiers!,} \lstinline!convertMessage! defined as follows. The
```

Comment:
Yes, it's a pity one needs to do things like this.  I think this is far from the only place where we have ugly line breaks before punctuation following `\lstinline`.

Reply:
Agreed, it would be good if there were a better solution for this.